p8Layouts/defaultPnorLayoutSingleSide.xml

```diff
@@ -266,6 +269,7 @@ Layout Description
         <physicalRegionSize>0x90000</physicalRegionSize>
         <side>A</side>
         <sha512Version/>
+        <readOnly/>
         <ecc/>
     </section>
     <section>
```

Comment:
What about all the other partitions that need to be read only, to bring the list to partity